staging/src/k8s.io/apiserver/pkg/endpoints/filters/authorization.go

```diff
@@ -55,8 +59,10 @@ func WithAuthorization(handler http.Handler, requestContextMapper request.Reques
 			return
 		}
 
+		gv := schema.GroupVersion{Group: attributes.GetAPIGroup(), Version: attributes.GetAPIVersion()}
+		gr := schema.GroupResource{Group: attributes.GetAPIGroup(), Resource: attributes.GetResource()}
 		glog.V(4).Infof("Forbidden: %#v, Reason: %q", req.RequestURI, reason)
-		responsewriters.Forbidden(attributes, w, req, reason)
```

Comment:
There is this `msg := sanitizer.Replace(forbiddenMessage(attributes))` line. We should preserve that. How do the error messages differ now from the old message?